src/components/CollapserRow.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState, useRef } from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+import Collapser from './Collapser';
+import { Icon } from '@newrelic/gatsby-theme-newrelic';
+
+const collapserIcon = (isOpen) => css`
+  margin-left: auto;
+  transition: transform 0.6s ease;
+  color: var(--accent-text-color);
+  ${isOpen &&
+  `
+transform: rotate(180deg);`}
+`;
+
+const CollapserRow = ({ title, id, openByDefault, children }) => {
+  const [isOpen, toggleOpen] = useState(openByDefault);
+  const [setHeight, setHeightState] = useState(isOpen ? 'auto' : '0px');
+
+  const content = useRef(null);
+  const toggleCollapser = () => {
+    toggleOpen(!isOpen);
+    setHeightState(isOpen ? '0px' : `${content.current.scrollHeight}px`);
+  };
+
+  return (
+    <Collapser>
```

Comment:
I see each row is wrapped by its on collapser. Do we want this? From a component design perspective, components tend to scale a bit better if components don't add their own surrounding whitespace (the callout components are the one exception to this). We want the parent components to layout their children so that they can space them accordingly. This ensures you can use a component in multiple contexts without having to rejigger the spacing each time you do.

Because of that, I think this surrounding `<Collapser>` should actually be removed and instead have the the consumer of the API use it.